.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: daily
```

Comment:
Daily might be a bit too frequent? Maybe weekly?

Reply:
The first run might be a bit overwhelming, but once versions are up-to-date, this is fine imo

It's what we do in other repos as well